supernode/daemon/mgr/dfgettask/manager.go

```diff
@@ -12,10 +13,15 @@ import (
 	dutil "github.com/dragonflyoss/Dragonfly/supernode/daemon/util"
 
 	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 )
 
 var _ mgr.DfgetTaskMgr = &Manager{}
 
+const (
+	keyJoinChar = "@"
```

Comment:
I am wondering if we could miss declaring this const variable. If codes below directly use "@", I think it is OK.

Reply:
I agree with you. Maybe it's OK. However, I think use a const variable is better to unify it in one place.